diff --git a/android/build.gradle b/android/build.gradle
index ae0b6dc717583c6bc1603a3ef1e4f284ccd72392..190644f1e95880812ee3938115772390e8e60f59 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -1,9 +1,9 @@
 buildscript {
     repositories {
+        mavenCentral()
         maven {
             url 'https://maven.google.com'
         }
-        jcenter()
     }
 
     dependencies {
@@ -36,6 +36,6 @@ repositories {
 }
 
 dependencies {
-    provided 'com.facebook.react:react-native:+'
-    compile group: 'commons-io', name: 'commons-io', version: '2.6'
+    implementation 'com.facebook.react:react-native:+'
+    implementation group: 'commons-io', name: 'commons-io', version: '2.6'
 }
